docs/tasks.md

```diff
@@ -10,12 +10,14 @@ A list of reusable Cumulus tasks. [Add your own.](adding-a-task.md)
 
 ## Tasks
 
-### [@cumulus/add-missing-file-checksums](https://github.com/nasa/cumulus/tree/master/tasks/add-missing-file-checksums)
+### [@cumulus/add-missing-file-checksums](https://github.com/nasa/cumulus/tree/master/tasks/add-missing-file-checksums#readme)
 
-Add checksums to granule files
+Add checksums to files in S3 which don't have one
```

Comment:
If we're changing the language here, we should probably change the language at: 

https://github.com/nasa/cumulus/tree/master/tasks/add-missing-file-checksums#readme

Reply:
Sorry, I haven't being working on `tasks.md`. I believe this came over from a merge from master. Do you want to add something to the `common use cases` section?  I'm not sure about checksums. I would have to do some research on that.

Reply:
I think in this instance we should change it back to `Add checksums to granule files`

Reply:
@Jkovarik Done.